symphony/content/content.systemauthors.php

```diff
@@ -339,10 +339,13 @@ public function __form()
             $label = Widget::Label(__('User Type'), null, 'column');
 
             $options = array(
-                array('author', false, __('Author')),
-                array('manager', $author->isManager(), __('Manager'))
+                array('author', false, __('Author'))
             );
 
+            if ($isOwner || Symphony::Author()->isDeveloper() || $author->isManager()) {
```

Comment:
Why `$isOwner` here ?

Reply:
It's used when `$isOwner` is a manager. The option is added so he can set himself as an author or stay a manager.